### DISCUSSION TITLE
Why visual studio blur the screen?

### DISCUSSION THREAD

**Discussion by AldoFlexo** [microsoft/vscode-discussions#166] [Extension Development QnA] [answered]
When I'm using visual studio in sometimes the screen start to blurried if I'm clicking other side of the screen or the second monitor, so if someone may help me, I really appreciate it.
![blur](https://user-images.githubusercontent.com/97918175/194791172-15eaef9e-c167-4b1f-9f1c-d05f965f0ba6.PNG)

**Comment by gjsjohnmurray** [comment, marked as answer]
This is off-topic for "Extension Development QnA". Please post at https://github.com/microsoft/vscode/issues
